docs/mod_tap.md

```diff
@@ -50,7 +50,9 @@ For convenience, QMK includes some Mod-Tap shortcuts to make common combinations
 
 ## Caveats
 
-Unfortunately, these keycodes cannot be used in Mod-Taps or Layer-Taps, since any modifiers specified in the keycode are ignored.
+Currently, all Mod-Tap keycodes (and also [`LT()`](feature_layers.md)) are limited to the [Basic Keycode set](keycodes_basic.md) for their *kc* argument, meaning you can't use keycodes like `LCTL()`, `KC_TILD`, or anything greater than `0xFF`. In the case of Mod-Tap, this is because QMK uses 16-bit keycodes, of which 3 bits are used for the function identifier, 1 bit is used for selecting right or left mods, and 4 bits are used to tell which mods are used, only 8 bits are left for the keycode. Because of this, the keycode used is limited to `0xFF` (0-255), which are the basic keycodes only. If at least one right-handed modifier is specified in a Mod-Tap, it will cause all modifiers specified to become right-handed, so it is not possible to mix and match the two.
```

Comment:
```suggestion
Currently, the `kc` argument of `MT()` is limited to the [Basic Keycode set](keycodes_basic.md), meaning you can't use keycodes like `LCTL()`, `KC_TILD`, or anything greater than `0xFF`. This is because QMK uses 16-bit keycodes, of which 3 bits are used for the function identifier, 1 bit for selecting right or left mods, and 4 bits to tell which mods are used, leaving only 8 bits for the keycode. Additionally, if at least one right-handed modifier is specified in a Mod-Tap, it will cause all modifiers specified to become right-handed, so it is not possible to mix and match the two.
```